holidays/countries/suriname.py

```diff
@@ -0,0 +1,227 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see AUTHORS.md file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from gettext import gettext as tr
+
+from holidays import HolidayBase
+from holidays.calendars import _CustomHinduHolidays, _CustomIslamicHolidays
+from holidays.calendars.gregorian import MAR, APR, MAY, JUN, JUL, AUG, SEP, OCT, NOV
+from holidays.groups import (
+    ChineseCalendarHolidays,
+    ChristianHolidays,
+    HinduCalendarHolidays,
+    InternationalHolidays,
+    IslamicHolidays,
+)
+
+
+class Suriname(
+    HolidayBase,
+    ChineseCalendarHolidays,
+    ChristianHolidays,
+    HinduCalendarHolidays,
+    InternationalHolidays,
+    IslamicHolidays,
+):
+    """Suriname holidays.
+
+    References:
+        * [Besluit Vrije Dagen 1971](https://web.archive.org/web/20250411201210/https://www.sris.sr/wp-content/uploads/2022/10/Besluit-Vrije-dagen-GB-1971-no-78.pdf)
+        * https://web.archive.org/web/20240416223555/https://gov.sr/wp-content/uploads/2022/09/arbeidswet.pdf
+        * [S.B. 2007 no. 98](https://web.archive.org/web/20250411204836/https://www.sris.sr/wp-content/uploads/2021/10/SB-2007-no-98-besluit-vrije-dagen.pdf)
+        * [S.B. 2012 no. 21](https://web.archive.org/web/20250411204754/https://www.sris.sr/wp-content/uploads/2021/10/SB-2012-no-21-besluit-vrije-dagen.pdf)
+        * [S.B. 2021 no. 27](https://web.archive.org/web/20250411210516/https://www.sris.sr/wp-content/uploads/2021/03/S.B.-2021-no.-27-houdende-nadere-wijziging-van-het-Besluit-Vrije-Dagen-1971.pdf)
+        * [S.B. 2024 no. 167](https://web.archive.org/web/20250411211129/https://www.sris.sr/wp-content/uploads/2025/01/S.B.-2024-no.-167-Algemene-termijnenwet.pdf)
+
+    Note:
+        The oldest decree available online that underpins the public holidays defined here
+        for Suriname is Besluit Vrije Dagen 1971 of April 22, 1971.
+
+        The S.B. 2024 no. 167 law only applies to prolongations of terms specified in contracts
+        and other similar legal documents, and does not apply to labor agreements (and days off).
+    """
+
+    country = "SR"
+    default_language = "nl"
+    # %s (estimated).
+    estimated_label = tr("%s (geschat)")
+    start_year = 1972
+    supported_languages = ("en_US", "nl")
+
+    def __init__(self, islamic_show_estimated: bool = False, *args, **kwargs):
```

Comment:
```suggestion
    def __init__(self, islamic_show_estimated: bool = True, *args, **kwargs):
```
Do we have any reason to assume that all the dates from Umm-Al-Qura are accurate for Suriname? 🙄

Reply:
Ah right, my bad.